### ISSUE TITLE
I can not access link outside my host laptop(Ubuntu)

### ISSUE THREAD

**Issue by marko-s6** [Unrud/remote-touchpad#69]
I am using Ubuntu 22.04.3 LTS, also I used the remote-touchpad on the same laptop a long time ago and it worked well. Now I can't access link from any of my devices on local network. But running it from the browser on laptop loads it just fine. I have already tried to bind port and then allow it on ufw in terminal, but it still does not work.

**Comment by KaptainAhoibrause** [comment]
Same thing for me

**Comment by jrutschi** [comment]
Do you use UFW? I had the same problem and solved it with the following rule, allowing TCP-traffic in the local network: `sudo ufw allow from 192.168.0.0/24`

**Comment by KaptainAhoibrause** [comment]
No, I use firewalld, but do you know the similar command there? Thanks, and sorry for late responding
